src/plugins/intel_gpu/tests/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -1686,9 +1690,9 @@ TEST(fully_connected_onednn_gpu, no_biases_int8) {
 
     ov::intel_gpu::ImplementationDesc fc_impl = { format::bfyx, "", impl_types::onednn };
 
-    ExecutionConfig cfg{ov::intel_gpu::queue_type(QueueTypes::in_order),
-                        ov::intel_gpu::force_implementations(ov::intel_gpu::ImplForcingMap{ {"fc_prim", fc_impl} })
-    };
+    ExecutionConfig cfg = get_test_default_config(engine);
+    cfg.set_property(ov::intel_gpu::queue_type(QueueTypes::in_order));
```

Comment:
no need to set in_order.

Reply:
As I know out-of-order queue-type will be supported on dGPU soon. Then the logic of setting in-order queue in get_test_default_config function should be removed. I think remaining this line will help the removal.

Reply:
Discussion : remove this line.
No reason to support and test in-order queue-type if oneDNN supports out-of-order.
it better remove in_order queue setting now. If thereis any special case for in-order queue test, then it should be handled in each unit-test when in-order queue is removed.